pom.xml

```diff
@@ -46,8 +46,9 @@
     </scm>
 
     <properties>
-        <jenkins.version>1.609.3</jenkins.version> <!-- but see Jenkins-Version override below -->
+        <jenkins.version>1.609.3</jenkins.version>
         <workflow.version>1.14</workflow.version>
+        <jenkins-test-harness.version>2.1</jenkins-test-harness.version>
```

Comment:
That is the default for this POM version, not necessary to override it unless you want to go newer.